src/main/webapp/dev/js/common/questionContrib.es6

```diff
@@ -23,6 +25,9 @@ function setDefaultContribQnVisibilityIfNeeded(questionNum) {
     $currentQuestionTable.find('input.visibilityCheckbox')
                          .filter('.recipientCheckbox')
                          .filter('[value="INSTRUCTORS"],[value="RECEIVER"]').prop('checked', true);
+    // Set dropdown button text
+    setVisibilityDropdownMenuText('Please select a visibility option <span class="caret"></span>',
```

Comment:
Maybe a better design could be to put that string in `const.es6`?

And BTW that value doesn't seem to come from a Java constant either. It appears to be hardcoded in Java too. Maybe I am missing something? :thinking: